locales/hi.json

```
@@ -1,1542 +1,1542 @@
+  "Welcome to Turtle Blocks": "टर्टल ब्लॉक्स में आपका स्वागत है",
+  "Turtle Blocks is a Logo-inspired turtle that draws colorful pictures with snap-together visual-programming blocks.": "टर्टल ब्लॉक्स एक लोगो-प्रेरित कछुआ है जो स्नैप-टूगेदर विजुअल-प्रोग्रामिंग ब्लॉक के साथ रंगीन तस्वीरें बनाता है।",
+  "The current version is": "वर्तमान संस्करण है",
+  "Click the run button to run the project in fast mode.": "तेज मोड में परियोजना को चलाने के लिए रन बटन पर क्लिक करें।",
+  "Stop the turtle.": "कछुआ को रोकें।",
+  "You can also type Alt-S to stop.": "आप रोकने के लिए Alt-S भी टाइप कर सकते हैं।",
+  "Welcome to Music Blocks": "म्यूजिक ब्लॉक में आपका स्वागत है",
+  "Music Blocks is a collection of tools for exploring fundamental musical concepts in a fun way.": "म्यूजिक ब्लॉक्स संगीत के मौलिक अवधारणाओं का अन्वेषण करने के लिए एक मजेदार तरीके से सुझाव देता है।",
+  "Meet Mr. Mouse!": "मिस्टर माउस से मिलें!",
+  "Mr Mouse is our Music Blocks conductor.": "मिस्टर माउस हमारे संगीत ब्लॉक्स कंडक्टर हैं।",
+  "Mr Mouse encourages you to explore Music Blocks.": "मिस्टर माउस आपको म्यूजिक ब्लॉक्स का अन्वेषण करने के लिए प्रेरित करता है।",
+  "Let us start our tour!": "आइए हमारा दौरा शुरू करें!",
+  "Guide": "मार्गदर्शिका",
+  "A detailed guide to Music Blocks is available.": "म्यूजिक ब्लॉक्स के लिए एक विस्तृत मार्गदर्शिका उपलब्ध है।",
+  "Stop the music (and the mice).": "संगीत को रोकें (और चूहे भी)।",
+  "Record your project as video.": "अपने परियोजना को एक वीडियो के रूप में रिकॉर्ड करें।",
+  "Toggle Fullscreen mode.": "पूर्णस्क्रीन मोड टॉगल करें",
+  "Initialize a new project.": "एक नई परियोजना की शुरुआत करें।",
+  "You can also load projects from the file system.": "आप फ़ाइल सिस्टम से भी परियोजनाएं लोड कर सकते हैं।",
+  "Save": "सहेजें",
+  "Save your project to a file.": "अपनी परियोजना को एक फ़ाइल में सहेजें।",
+  "Save graphics from your project to as SVG.": "अपनी परियोजना से ग्राफिक्स को एसवीजी के रूप में सहेजें।",
+  "Save graphics from your project as PNG.": "अपनी परियोजना से ग्राफिक्स को पीएनजी के रूप में सहेजें।",
+  "Save block artwork as an SVG file.": "ब्लॉक कला को एसवीजी फ़ाइल के रूप में सहेजें।",
+  "Save sheet music as ABC, Lilypond or MusicXML": "शीट संगीत को ABC, Lilypond या MusicXML के रूप में सहेजें",
+  "Save block artwork as SVG or PNG": "ब्लॉक आर्टवर्क को SVG या PNG के रूप में सहेजें",
+  "Load samples from server": "सर्वर से नमूना लोड करें",
+  "This button opens a viewer for loading example projects.": "यह बटन नमूने देखने के लिए एक दर्शक खोलता है।",
+  "Expand/collapse option toolbar": "विकल्प टूलबार को विस्तृत/संकुचित करें",
+  "Click this button to expand or collapse the auxillary toolbar.": "सहायक उपकरण पट्टी को विस्तृत या संक्षिप्त करने के लिए यह बटन क्लिक करें.",
+  "Displays help messages for the main and auxiliary toolbar, right-click contextual menu for blocks and canvas, and palettes.": "मुख्य और सहायक टूलबार, ब्लॉक्स और कैनवास के राइट-क्लिक संदर्भ मेनू, और पैलेट्स के लिए सहायता संदेश दिखाता है।",

```

Perhaps transliterating menu as how it is pronounced might be better? मेन्यू ?

and instead of संदर्भ as a translation, how about प्रासंगिक as translation of contextual? 

```suggestion
  "Displays help messages for the main and auxiliary toolbar, right-click contextual menu for blocks and canvas, and palettes.": "मुख्य और सहायक टूलबार, ब्लॉक्स और कैनवास के राइट-क्लिक संदर्भ मेन्यू, और पैलेट्स के लिए सहायता संदेश दिखाता है।",

```